### ISSUE TITLE
Client side example

### ISSUE THREAD

**Issue by joserocha3** [coformatique/prisma-auth0-starter#1]
Great starter! This is exactly what the Prisma community is missing. One issue I have is that I am not able to make calls to the server from the client side. I am setting the `Authorization` header with `Bearer id_token_here`. I get the following error:

```
Token error: UnauthorizedError: jwt audience invalid. expected: https://pablo-personal.auth0.com/api/v2/
```

Are you working on a client side companion repo? That would be extremely useful.

**Comment by picosam** [comment]
Hey @joserocha3! Thank you, I wouldn't have done it without the community support... but indeed, it's not done yet! When it's ready, I'll put the word out on the Graphcool Forum.

That said, yes, I do have an Angular client working with this. Are you using [Auth0.js](https://auth0.com/docs/libraries/auth0js)? What value are you currently assigning to `audience`?

**Comment by joserocha3** [comment]
Thanks for the quick reply @picosam. I am using Auth0.js and my audience is `https://pablo-personal.auth0.com/api/v2/`. That is reflected in the Auth0 dashboard as well.

**Comment by picosam** [comment]
Right; and is your `AUTH0_DOMAIN` in your `.env` file set to `pablo-personal.auth0.com`?

**Comment by joserocha3** [comment]
Yes sir, it is `AUTH0_DOMAIN="pablo-personal.auth0.com"`

**Comment by picosam** [comment]
Alright; could you please check back on Sunday after I've added the unit tests? I can't figure out what may be the issue right now. The whole idea is that the audience set on your GraphQL server would be the same as that you're using on your client app.

**Comment by picosam** [comment]
For now, if you have time, please check if the latest version I just pushed gives you some more hints. Note: I added `AUTH0_AUDIENCE` in `.env` and I added a couple of links to `README.md` that you should definitely read if you haven't already. Stay tuned!

**Comment by picosam** [comment]
FYI: https://github.com/coformatique/prisma-auth0-angular-starter
Happy Easter!
